content/sensu-go/5.19/sensuctl/reference.md

```diff
@@ -648,6 +648,109 @@ See the [RBAC reference][21] for information about using access control with nam
 
 See the [RBAC reference][22] for information about local user management with sensuctl.
 
+#### sensuctl prune
+
+{{% notice important %}}
+**IMPORTANT**: `sensuctl prune` is an alpha feature in release 5.19.0 and may include breaking changes.
+{{% /notice %}}
+
+**COMMERCIAL FEATURE**: Access sensuctl pruning in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][30].
+
+The `sensuctl prune` subcommand allows you to delete resources that do not appear in a given set of Sensu objects (called a "configuration") from a from a file, URL, or STDIN.
+For example, you can use `sensuctl create` to to apply a new configuration, then use `sensuctl prune` to prune unneeded resources, resources that were created by a specific user or that include a specific label selector, and more.
+
+`sensuctl prune` can only delete resources that have the label `sensu.io/managed_by: sensuctl`, which Sensu automatically adds to all resources created with sensuctl.
+This means you can only use `sensuctl prune` for delete resources that were created with sensuctl.
```

Comment:
You would know better than me in terms of grammar, is `for` or `to` the correct usage here?
```suggestion
This means you can only use `sensuctl prune` to delete resources that were created with sensuctl.
```

Reply:
no you are correct! I probably changed "deleting" to "delete" and forgot to fix the "for" 
Thank you for checking